docs/tutorials/transforms.ipynb

```diff
@@ -260,30 +233,30 @@
   },
   {
    "cell_type": "code",
+   "execution_count": 5,
    "metadata": {
     "colab": {
      "base_uri": "https://localhost:8080/"
     },
     "id": "VHVgiNA4t5Tl",
     "outputId": "64cb16f2-e85e-4f36-a046-129098625b40"
    },
-   "source": [
-    "dataset = EuroSAT(download=True)\n",
-    "dataloader = DataLoader(dataset, batch_size=4, shuffle=True, num_workers=2)\n",
-    "dataloader = iter(dataloader)\n",
-    "print(f\"Number of images in dataset: {len(dataset)}\")\n",
-    "print(f\"Dataset Classes: {dataset.classes}\")"
-   ],
-   "execution_count": 4,
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
+     "output_type": "stream",
      "text": [
-      "Number of images in dataset: 27000\n",
+      "Number of images in dataset: 16200\n",
```

Comment:
I'm guessing this changed because the original dataset had no train/val/test splits and these were added after the tutorial was written?